### DISCUSSION TITLE
How to transfer all repositories to an organization?

### DISCUSSION THREAD

**Discussion by ghost** [community/community#66360] [Repositories] [answered]
### Select Topic Area

Question

### Body

Hello everyone.

I would like to know how to transfer all repositories to an organization. How to transfer all repositories to an organization?  Is there any way to automate the transfer of x amount of repositories from my personal GitHub profile to an organization?

**Comment by seyLu** [comment, marked as answer]
#### Hi @solarsbeans 

It is possible to automate it via Github REST API:
1. [Create org](https://docs.github.com/en/rest/repos/repos?apiVersion=2022-11-28#create-an-organization-repository)
2. [List all repos from user](https://docs.github.com/en/rest/repos/repos?apiVersion=2022-11-28#list-repositories-for-a-user) 
3. [Transfer repo ownership from user to org](https://docs.github.com/en/rest/repos/repos?apiVersion=2022-11-28#transfer-a-repository)

**Reply by ghost** [reply]
Hi seyLu, how are you?

Thank you so much for answering my question. I will access and read the documentation to see how I do this: migrate repositories in an x quantity through the GitHub API in rest.
